README.md

```diff
@@ -238,9 +212,11 @@ After installing the plugin and syncing your Android Studio project with Gradle,
             // 4. Instantiate an instance of the CodePush runtime, using the right deployment key
             this._codePush = new CodePush("0dsIDongIcoH0mqAmoR0CYb5FhBZNy1w4Bf-l", this, BuildConfig.DEBUG);
```

Comment:
Could we change this constructor to take a ReactActvitty instance instead of just an Activity? Then we could remove the third parameter and determine debug mode by simply calling the ReactActivitiy's method.